sql/src/main/scala/org/apache/spark/sql/sedona_sql/strategy/join/TraitJoinQueryExec.scala

```diff
@@ -61,13 +61,9 @@ trait TraitJoinQueryExec extends TraitJoinQueryBase {
     if (sedonaConf.getJoinApproximateTotalCount == -1) {
       if (sedonaConf.getJoinSparitionDominantSide == JoinSparitionDominantSide.LEFT) {
         leftShapes.analyze()
-        sedonaConf.setJoinApproximateTotalCount(leftShapes.approximateTotalCount)
```

Comment:
These where set but never read

Reply:
I also forgot why it was set but not used...